src/scrapers/visa-cal.js

```diff
@@ -10,7 +10,8 @@ import {
   SHEKEL_CURRENCY_SYMBOL,
   SHEKEL_CURRENCY,
   DOLLAR_CURRENCY_SYMBOL,
-  DOLLAR_CURRENCY } from '../constants';
+  DOLLAR_CURRENCY,
+} from '../constants';
```

Comment:
this change was done since eslint blocked my commits until I fixed the broken eslint rule.